src/docs/03-Code Guidelines/06-Changelog.md

```diff
@@ -0,0 +1,16 @@
+<div class="frctl-example">
+  <div class="changelog">

```

Comment:
html should be 4 spaces indentation. Even if it's .hbs or .md files wouldn't you agree?